README.md

```diff
@@ -1,38 +1,33 @@
 # panolint-ruby
 
 [![Build Status](https://github.com/panorama-ed/panolint-ruby/workflows/Main/badge.svg)](https://github.com/panorama-ed/panolint-ruby/actions)
-[![Gem Version](https://img.shields.io/gem/v/panolint-ruby.svg)](https://github.com/panorama-ed/panolint-ruby)
 
-A small gem containing rules for linting Ruby code at Panorama Education.
+A small gem containing RuboCop linting rules for Panorama Education code.
 
 ## Installation
 
 Add this line to your application's Gemfile:
 
-```ruby
-gem 'panolint-ruby'
+```
+gem "panolint-ruby", git: "https://github.com/panorama-ed/panolint-ruby.git", branch: "main"
 ```
 
 And then execute:
 
     $ bundle
 
-Or install it yourself as:
-
-    $ gem install panolint-ruby
+Do not install this gem from RubyGems, as updates are only pushed to this repo. We have only pushed an initial version to RubyGems to avoid supply chain attacks from malicious squatters.
```

Comment:
Slight preference for bolding `Do not install this gem from RubyGems` but I don't feel strongly.